mol2 test failure on py27

================================== FAILURES ===================================
```
________________________ test_load_all_mol2[adp.mol2] _________________________
mol2_file = 'adp.mol2', get_fn = <function _get_fn at 0x089200B0>
    @pytest.mark.parametrize('mol2_file', [('li.mol2'),
    ('lysozyme-ligand-tripos.mol2'), ('imatinib.mol2'),
    ('status-bits.mol2'),('adp.mol2')])
    def test_load_all_mol2(mol2_file, get_fn):
>       trj = md.load_mol2(get_fn(mol2_file))
tests\test_mol2.py:156: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
..\_test_env\lib\site-packages\mdtraj\formats\mol2.py:130: in load_mol2
    top = Topology.from_dataframe(atoms_mdtraj, bonds_mdtraj)
..\_test_env\lib\site-packages\mdtraj\core\topology.py:510: in from_dataframe
    a = Atom(atom['name'], elem.get_by_symbol(atom['element']),
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
symbol = 'O3'
    def get_by_symbol(symbol):
        s = symbol.strip().upper()
>       return Element._elements_by_symbol[s]
E       KeyError: 'O3'
..\_test_env\lib\site-packages\mdtraj\core\element.py:230: KeyError
=========================== short test summary info ===========================
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -7,13 +7,9 @@ environment:
     PYTHONUNBUFFERED: 1
 
   matrix:
-    - PYTHON: "C:\\Miniconda36"
-      CONDA_PY: "27"
     - PYTHON: "C:\\Miniconda36-x64"
       CONDA_PY: "27"
       ARCH: "64"
-    - PYTHON: "C:\\Miniconda36"
-      CONDA_PY: "36"
     - PYTHON: "C:\\Miniconda3-x64"
       CONDA_PY: "36"
       ARCH: "64"

--- a/devtools/conda-recipe/meta.yaml
+++ b/devtools/conda-recipe/meta.yaml
@@ -41,7 +41,7 @@ test:
     - shiftx2  # [linux and py2k]
     # do not install cpptraj on osx, because the current version from omnia is linked against an old runtime.
     # and not avail on win
-    - cpptraj  # [not (osx or win32) ]
+    #- cpptraj  # [not (osx or win32) ]
     - nbformat
     - ipykernel
     - scikit-learn

--- a/examples/test_examples.py
+++ b/examples/test_examples.py
@@ -15,7 +15,7 @@ FLAKEY_LIST = ['centroids.ipynb', 'native-contact.ipynb']
 TIMEOUT = 60  # seconds
 
 test_dir = os.path.dirname(os.path.abspath(__file__))
-examples = [pytest.param(pytest.mark.flaky(fn)) if fn in FLAKEY_LIST else fn
+examples = [pytest.param(fn, marks=pytest.mark.flaky()) if fn in FLAKEY_LIST else fn
             for fn in os.listdir(test_dir) if fn.endswith('.ipynb')]
 
 

--- a/examples/test_examples.py
+++ b/examples/test_examples.py
@@ -11,7 +11,7 @@ import pytest
 from jupyter_client import KernelManager
 from six.moves.queue import Empty
 
-FLAKEY_LIST = ['centroids.ipynb', 'native-contact.ipynb']
+FLAKEY_LIST = ['centroids.ipynb', 'native-contact.ipynb', 'hbonds.ipynb']
 TIMEOUT = 60  # seconds
 
 test_dir = os.path.dirname(os.path.abspath(__file__))

--- a/examples/test_examples.py
+++ b/examples/test_examples.py
@@ -15,7 +15,7 @@ FLAKEY_LIST = ['centroids.ipynb', 'native-contact.ipynb']
 TIMEOUT = 60  # seconds
 
 test_dir = os.path.dirname(os.path.abspath(__file__))
-examples = [pytest.mark.flaky(fn) if fn in FLAKEY_LIST else fn
+examples = [pytest.param(pytest.mark.flaky(fn)) if fn in FLAKEY_LIST else fn
             for fn in os.listdir(test_dir) if fn.endswith('.ipynb')]
 
 

--- a/mdtraj/formats/mol2.py
+++ b/mdtraj/formats/mol2.py
@@ -99,7 +99,23 @@ def load_mol2(filename):
     # If this is a sybyl mol2, there should be NAN (null) values
     if atoms_mdtraj.element.isnull().any():
         # If this is a sybyl mol2, I think this works generally.
-        atoms_mdtraj["element"] = atoms.atype.apply(lambda x: x.split(".")[0])
+        # Argument x is being passed as a list with only one element.
+        def to_element(x):
+            if isinstance(x, (list, tuple)):
+                assert len(x) == 1
+                x = x[0]
+
+            if '.' in x:  # orbital-hybridizations in SYBL
+                return x.split('.')[0]
+            try:
+                # check if we can convert the whole str to an Element,
+                # if not, we only pass the first letter.
+                from mdtraj.core.element import Element
+                Element.getBySymbol(x)
+            except KeyError:
+                return to_element(x[0])
+            return x
+        atoms_mdtraj["element"] = atoms.atype.apply(to_element)
 
     atoms_mdtraj["resSeq"] = np.ones(len(atoms), 'int')
     atoms_mdtraj["chainID"] = np.ones(len(atoms), 'int')

--- a/mdtraj/formats/mol2.py
+++ b/mdtraj/formats/mol2.py
@@ -200,14 +200,14 @@ def mol2_to_dataframes(filename):
         csv.writelines(data["@<TRIPOS>BOND\n"][1:])
         csv.seek(0)
         bonds_frame = pd.read_table(csv, names=["bond_id", "id0", "id1", "bond_type"],
-            index_col=0, header=None, sep=r"\s*", engine='python')
+            index_col=0, header=None, sep="\s+", engine='python')
     else:
         bonds_frame = None
 
     csv = StringIO()
     csv.writelines(data["@<TRIPOS>ATOM\n"][1:])
     csv.seek(0)
-    atoms_frame = pd.read_csv(csv, sep=r"\s*", engine='python',  header=None)
+    atoms_frame = pd.read_csv(csv, sep="\s+", engine='python',  header=None)
     ncols = atoms_frame.shape[1]
     names=["serial", "name", "x", "y", "z", "atype", "code", "resName", "charge", "status"]
     atoms_frame.columns = names[:ncols]

